### PR TITLE
Proxy Port Fixes 

### PR DESCRIPTION
**Pull request by bitmold**
This branch is dependent on https://github.com/guardianproject/tor-android/pull/84 being released in tor-android and should not be merged until that is released 
Fixes: #708 and #709

### CODE REVIEW

**Comment by bitmold** [comment]
Also fixes #707

File: orbotservice/src/main/java/org/torproject/android/service/OrbotService.java [hunk 1]
```diff
@@ -511,12 +514,14 @@ private File updateTorrcCustomFile() throws IOException {
...
-        extraLines.append("TestSocks 0").append('\n');
-
-        if (Prefs.openProxyOnAllInterfaces())
-            extraLines.append("SocksListenAddress 0.0.0.0").append('\n');
+        if (!Prefs.openProxyOnAllInterfaces()) {
+            extraLines.append("SOCKSPort ").append(socksPortPref).append(isolate).append(ipv6Pref).append('\n');
+            extraLines.append("SafeSocks 0").append('\n');
+            extraLines.append("TestSocks 0").append('\n');
+        } else {
+            extraLines.append("SOCKSPort 0.0.0.0:").append(socksPortPref).append("\n");
```
Comment (SkewedZeppelin) [inline orbotservice/src/main/java/org/torproject/android/service/OrbotService.java:522]: should isolate/ipv6 be added here too?

Comment (bitmold) [reply orbotservice/src/main/java/org/torproject/android/service/OrbotService.java]: yes, thank you. added

**Comment by syphyr** [comment]
It looks like R.id.tvSnowflake is undefined

**Comment by syphyr** [comment]
This fixes the problem: https://github.com/syphyr/orbot/commit/63c0060a38d980fbcab73eb11d28fd0976325803

**Comment by bitmold** [comment]
Thanks @syphyr , I just forgot to commit the XML file. I'll throw it up next time I'm online

**Comment by syphyr** [comment]
One other thing I noticed is that the version string is only showing up for obfs4 in "About" screen.

**Comment by bitmold** [comment]
Odd, I was seeing both versions. Will revisit when I commit the missing XML file ...

**Comment by SkewedZeppelin** [comment]
eh, https://github.com/guardianproject/tor-android/pull/84 was merged and released?

weren't these supposed to be together?

**Pull request merged**